### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by richadivami**
# TITLE

## Description
Summary of changes

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [Jira issue #](fill-in.org)

### CODE REVIEW

File: app/cdap/components/ConnectionList/index.tsx [hunk 1]
```diff
@@ -204,8 +204,9 @@ export default function ConnectionList() {
       <SubHeader />
       <SelectDatasetWrapper>
         {dataForTabs.map((each, index) => {
-          if (each.data.filter((el) => el.connectionId).length) {
-            connectionId = each.data.filter((el) => el.connectionId)[0].connectionId;
+          const connectionIDRequired = each.data.filter((el) => el.connectionId);
```
Comment (adityadivami) [inline app/cdap/components/ConnectionList/index.tsx]: connectionIDRequired
change to connectionIdRequired

Comment (richadivami) [reply app/cdap/components/ConnectionList/index.tsx]: Done

File: app/cdap/components/GridTable/components/GridKPICell/index.tsx [hunk 2]
```diff
@@ -33,8 +33,11 @@ export default function GridKPICell({ metricData }) {
             <Box className={classes.KPICell} key={eachValue.label}>
               <Typography className={classes.label}>{eachValue.label}</Typography>
               <Typography
-                className={classes.count}
-                style={eachValue.label == 'Missing/Null' ? { color: 'red' } : { color: '#000' }}
+                className={
+                  eachValue.label == 'Missing/Null'
```
Comment (adityadivami) [inline app/cdap/components/GridTable/components/GridKPICell/index.tsx]: move strings to constants relative to the component

File: app/cdap/components/GridTable/components/GridKPICell/styles.ts [hunk 3]
```diff
@@ -50,4 +50,10 @@ export const useGridKPICellStyles = makeStyles({
     fontWeight: 400,
     color: 'red',
   },
+  missingClass: {
+    color: 'red',
```
Comment (adityadivami) [inline app/cdap/components/GridTable/components/GridKPICell/styles.ts]: import purple from '@material-ui/core/colors/purple';
import red from '@material-ui/core/colors/red';

const primary = red[500]; // #f44336


use colors from mui color palette

File: app/cdap/components/GridTable/components/GridKPICell/index.tsx (hunk 2 above)
Comment (richadivami) [reply app/cdap/components/GridTable/components/GridKPICell/index.tsx]: Done

File: app/cdap/components/GridTable/components/GridKPICell/styles.ts (hunk 3 above)
Comment (richadivami) [reply app/cdap/components/GridTable/components/GridKPICell/styles.ts]: Done

**Pull request merged**